File: 29- চিনহ পরিচয়.c
#include<stdio.h>
int main()
{
    int t;
    char ch;
    scanf("%d",&t);
    while(t--)
    {
        getchar();
        scanf("%c",&ch);
        if(ch >='a' && ch<='z')
            printf("Lowercase Character\n");
        else if(ch >='A' && ch<='Z')
            printf("Uppercase Character\n");
        else if(ch>='0' && ch<='9')
            printf("Numerical Digit\n");
        else
            printf("Special Character\n");


    }

    return 0;


}

File: 29- চিনহ পরিচয়.py
t = int(input())

li = []

while t!=0:
    s = input()
    li.append(s)
    t-=1

for i in li:
    if i>='A' and i<='Z':
        print("Uppercase Character")
    elif i>='a' and i<='z':
        print("Lowercase Character")
    elif i>='0' and i<='9':
        print("Numerical Digit")
    else:
        print("Special Character")
